docs/docs/cross-origin-errors.md

```diff
@@ -0,0 +1,33 @@
+---
+id: cross-origin-errors
+title: Cross-origin Errors
+permalink: docs/cross-origin-errors.html
+---
+
+> Note:
+>
+> The following section applies only to the development mode of React. Error handling in production mode is done with regular try/catch statements.
+
+In [development mode](https://facebook.github.io/react/docs/optimizing-performance.html), React uses a global `error` event handler to preserve the "Pause on exceptions" behavior of browser DevTools. It also logs errors to the developer console.
+
+If an error is thrown from a [different origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) the browser will mask its details and React will not be able to log the original error message. This is a security precaution taken by browsers to avoid leaking sensitive information.
+
+If you trust the scripts you're running you can ask the browser to bypass the same-origin policy in a secure way. Doing this can help simplify the development/debugging process. Below are a some common causes of cross-origin errors and ways to address them.
```

Comment:
Sorry, missed this before you put up the PR: You should always trust the scripts you're running. :) This isn't actually the relevant characteristic here I think. (More strictly, it's "if the scripts you're running trust you" but I don't think that's clearer.)

Can this be "If your server is set up to deliver scripts with the proper headers" or something like that?